src/swarm-lang/Swarm/Language/Parser/Value.hs

```diff
@@ -20,7 +24,8 @@ import Text.Megaparsec qualified as MP
 
 readValue :: Type -> Text -> Maybe Value
 readValue ty txt = do
-  s <- eitherToMaybe $ readNonemptyTerm txt
+  txt' <- T.stripPrefix "paper:" txt <|> pure txt
```

Comment:
Can we avoid hard-coding this to `paper`?

Reply:
Good catch, I missed this.  However, the right way to generalize it is not clear to me.  Since this lives in `swarm-lang` we can't get a list of entities and their properties.  A simple idea would be to just drop everything until the first colon, but colons can legitimately show up in `Text` values, for example `read "\"hi: there\"" : Text` should still work...

Reply:
OK, I hacked something together that strips up to the first colon, UNLESS the first colon is preceded by a double quote.  Ugly and possibly not quite correct but I think it will work.